services-api/src/main/java/io/scalecube/services/transport/api/ServiceTransport.java

```diff
@@ -46,5 +46,5 @@ static ServiceTransport getTransport() {
    * @param executorService transport executor service
    * @return shutdown signal
    */
-  Mono<Void> shutdown(ExecutorService executorService);
+  Mono shutdown(ExecutorService executorService);
```

Comment:
Why `Void` is removed? What are we expecting from return type `Mono`?

Reply:
@dmytro-lazebnyi good point, I will set it back